src/org/parosproxy/paros/network/ConnectionParam.java

```diff
@@ -223,6 +225,13 @@ protected void parse() {
         }
         
         try {
+            this.httpStateEnabled = getConfig().getBoolean(HTTP_STATE_ENABLED, true);
+        } catch (ConversionException e) {
+            log.error("Error while loading the option httpStateEnabled: " + e.getMessage(), e);
+        }
+        
+        

```

Comment:
Can't wait for https://github.com/zaproxy/zap-api-java/pull/32 to come in place ...